drupal/sites/default/boinc/modules/boincwork/boincwork.module

```diff
@@ -162,6 +162,32 @@ function boincwork_menu() {
     'access arguments' => array('access content'),
     'type' => MENU_CALLBACK
   );
+  // @todo - change URL of routes
```

Comment:
Is this still a todo? If yes, can you elaborate why and how the URLs should be changed?